app-editors/vscode/files/vscode-url-handler.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Visual Studio Code - URL Handler
+Comment=Code Editing. Redefined.
+GenericName=Text Editor
+Exec=/usr/bin/code --open-url %U
```

Comment:
We renamed this to `vscode`, well, let's be consistent.

Reply:
I just forgot to check that file x)

Any thoughts on the naming `vscode-bin` ? Can we rename the package later on ? Like is there something in EBUILDs to tell portage that a package just changed its name ? I suppose that exists. I am thinking if ever built-from-source vscode gets upstreamed, people will easily see that `vscode-bin` is a binary package whereas `vscode-oss` isn't.